yb-voyager/cmd/exportData.go

```diff
@@ -307,12 +326,14 @@ func checkDataDirs() {
 	exportDataDir := filepath.Join(exportDir, "data")
 	flagFilePath := filepath.Join(exportDir, "metainfo", "flags", "exportDataDone")
 	propertiesFilePath := filepath.Join(exportDir, "metainfo", "conf", "application.properties")
+	debeziumLogsPath := filepath.Join(exportDir, "logs", "debezium.log")
 	dfdFilePath := exportDir + datafile.DESCRIPTOR_PATH
 	if startClean {
 		utils.CleanDir(exportDataDir)
 		os.Remove(flagFilePath)
 		os.Remove(dfdFilePath)
 		os.Remove(propertiesFilePath)
+		os.Remove(debeziumLogsPath)
```

Comment:
Do not remove log files please :-)

Reply:
This is on start-clean! Do we not do remove voyager-logs on start-clean? 😅 . By default, if not removed, debezium starts creating multiple log files like debezium.log.1, debezium.log.2, etc. Is that fine?

Reply:
We don't remove voyager log files on --start-clean.

Can you make debezium to append the logs in the debezium.log? Or at least restrict the number of log files by deleting older ones?

Reply:
As discussed, reverting to older solution of a single log output of console, and stdout,stderr redirected to file.